bridge/client/app/_components/ktb-webhook-settings/ktb-webhook-settings.component.html

```diff
@@ -21,14 +21,17 @@ <h2>Webhook configuration</h2>
       <dt-error *ngIf="getFormControl('url').errors?.required">URL must not be empty</dt-error>
       <dt-error *ngIf="getFormControl('url').errors?.url">URL must start with http(s)://</dt-error>
       <div dtSuffix>
-        <ng-container
-          [ngTemplateOutlet]="secretButton"
-          [ngTemplateOutletContext]="{
-            $implicit: undefined,
-            controlName: 'url',
-            selectionStart: urlInput.selectionStart
-          }"
-        ></ng-container>
+        <ktb-variable-selector
```

Comment:
For this component, I think it would make sense to put this into an ng-template and reference it at needed places to avoid duplications. Also if we need to change text etc. we don't have to touch 3 places.